src/dep/libs/lib7z/7zFile.h

```diff
@@ -4,7 +4,7 @@
 #ifndef __7Z_FILE_H
 #define __7Z_FILE_H
 
-#if defined(_WIN32) && !defined(__LIBRETRO__)
+#ifdef _WIN32

```

Comment:
@barbudreadmon, what about this?
@loggan08, what are the reasons for removing the libretro if's (here and below in 7zTypes.h)?

Reply:
> Most likely @Loggan08 overwrited the lib7z library without looking if there were hacks in the previous one.
> Furthermore, i looked at this lib7z upgrade, it seems utterly pointless ? The only changes, besides removing my libretro hacks, are about bumping dates and moving/removing comments around...

It's the same SDK version, i tried bumping it, but it requires a rewrite of un7z.cpp